### PR TITLE
Add function for returning default os params

### PR DESCRIPTION
**Pull request by igchor**
To limit boilerplate in tests and user code.

### CODE REVIEW

**Review by bratpiorka — APPROVED** [review]

File: src/memory_targets/memory_target_numa.c
Comment (lukaszstolarczuk) [inline src/memory_targets/memory_target_numa.c:52]: you could "save" these TODOs  (perhaps move them or file an issue for them), unless it's already been resolved 😉

File: test/provider_os_memory.cpp
Comment (lukaszstolarczuk) [inline test/provider_os_memory.cpp:36]: traces set here to 1 (not sure if relevant, though). Please, confirm it's not just omitted by mistake

File: test/provider_os_memory_config.cpp
Comment (lukaszstolarczuk) [inline test/provider_os_memory_config.cpp:33]: .

File: src/memory_targets/memory_target_numa.c
Comment (igchor) [reply src/memory_targets/memory_target_numa.c:52]: Those TODO are not longer relevant actually. This entire initialization code was just copied from somewhere with those TODOs but in this file, we specifically set maxnode to number of memory targets.

Comment (igchor) [reply src/memory_targets/memory_target_numa.c:52]: And in tests, we already have a mechanism for finding maxnode.

File: test/provider_os_memory.cpp
Comment (igchor) [reply test/provider_os_memory.cpp:36]: It was intentional. Some ENV variable should control the traces and we should not hardcode it in tests to 1.

**Review by lukaszstolarczuk — COMMENTED** [review]
👍

**Comment by bratpiorka** [comment]
@lukaszstolarczuk @ldorau I need second approval before I merge these changes

**Review by ldorau — APPROVED** [review]

**Pull request merged**
